### PR TITLE
Update ProseMirror-Tables to 1.2.2

### PR DESCRIPTION
**Pull request by sven-coppers**
I am struggling with a bug when resizing tables in TipTap. This exact bug seems to be fixed in ProseMirror-Tables: https://github.com/ProseMirror/prosemirror-tables/commit/23c79aeb323ba5f5498142b4a438f9b3a4729844

The first version of ProseMirror-table to include this fix is 1.2.2

### CODE REVIEW

File: packages/extension-table/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@tiptap/core": "^2.0.0-beta.1"
   },
   "dependencies": {
-    "@_ueberdosis/prosemirror-tables": "1.1.3",
+    "@_ueberdosis/prosemirror-tables": "1.2.2",
```
Comment (bdbch) [inline packages/extension-table/package.json:27]: We were using a temporary package here to fix typescript dependencies a bit back.
`@_ueberdosis/prosemirror-tables` won't work as we don't have that version published on npm.

I guess you wanted to change to `prosemirror-tables@1.2.2`? if you do this, can you test it and let me know? I'll take another look at your PR then.

**Review by bdbch — CHANGES_REQUESTED** [review]

File: packages/extension-table/package.json (hunk 1 above)
Comment (sven-coppers) [reply packages/extension-table/package.json]: That is indeed exactly what I meant. I have made the changes, tested it locally, and pushed it to GitHub.

image
image

For some reason, the GitHub tests keep failing though.

Comment (bdbch) [reply packages/extension-table/package.json]: I'll take a look at them! :) Thanks for reacting so quickly!

Comment (sven-coppers) [reply packages/extension-table/package.json]: Thanks to you as well!

I had no idea how to make the code pass the GitHub "building phase", so you might want to ignore my third commit (c8be86c). Cypress keeps throwing timeouts.

Hope it gets resolved soon 🤞

**Comment by sven-coppers** [comment]
I can't get it to work locally or on GitHub. No idea how the tests passed on my computer.

Anyway, for me, the update is no longer needed. I am closing this pull request since the code in it does not work anyway.

Anyone who is interested in an update to ProseMirror-Tables@1.2.2 can watch https://github.com/ueberdosis/tiptap/issues/3083

**Pull request closed without merging**